src/procedure/nodes/sizeFactor.h

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// Copyright (c) 2023 Team Dissolve and contributors
+
+#pragma once
+
+#include "procedure/nodes/node.h"
+class SizeFactorProcedureNode : public ProcedureNode
+{
+    public:
+    SizeFactorProcedureNode();
+    ~SizeFactorProcedureNode() override = default;
+
+    /*
+     * Identity
+     */
+    public:
+    // Return whether a name for the node must be provided
+    bool mustBeNamed() const override;
+
+    /*
+     * Node Data
+     */
+    private:
+    NodeValue sizeFactor_{1.0};
```

Comment:
I think we can default to `10.0` here since adding the node into the procedure implies you want to do the scaling, and `10.0` is what most people will go for.
```suggestion
    NodeValue sizeFactor_{10.0};
```